hnginternship5/lucid
# Minor bug fix- portfolio page

### TITLE OF PR
Removed dummy portfolio

### PURPOSE OF PR
To  ascertain responsiveness of new portfolio files

### FEATURES IMPLEMENTED
Deleted dummy portfolio card in portfolio.html
Deleted create-portfolio.html

### HOW TO TEST
Go to /portfolio and attempt to create a portfolio file. It should work as expected.

### Add page screen shot from your local
![1](https://user-images.githubusercontent.com/40725858/57696701-4ee79200-7649-11e9-835a-387dac22d956.png)
![2](https://user-images.githubusercontent.com/40725858/57696703-4f802880-7649-11e9-904d-42529458a40d.png)
